util/retry/retry.go

```diff
@@ -78,13 +78,17 @@ type Options struct {
 // returns an error.
 func WithBackoff(opts Options, fn func() (Status, error)) error {
 	backoff := opts.Backoff
+	tag := opts.Tag
+	if tag == "" {
+		tag = "invocation"
```

Comment:
"?" looks weird in the output. I had tried a few options and I'd rather leave it as is.